tests/network_test.cpp

```diff
@@ -189,7 +189,12 @@ TEST_F(NetworkTest, DISABLED_sync_large_pbft_block) {
 
   auto pbft_blocks1 = nodes[0]->getDB()->getPbftBlock(1);
   auto pbft_blocks2 = nodes2[0]->getDB()->getPbftBlock(1);
-  EXPECT_EQ(pbft_blocks1, pbft_blocks2);
+  EXPECT_EQ(pbft_blocks1->rlp(true), pbft_blocks2->rlp(true));
```

Comment:
you are modifying disabled test, so if this is the fix for it, maybe you should enabled it, if not do we need this test?

Reply:
I will move all that changes to a separate branch where this is fixed